connector/src/main/scala/com/microsoft/kusto/spark/utils/CslCommandsGenerator.scala

```diff
@@ -63,11 +63,23 @@ private[kusto] object CslCommandsGenerator {
        |  distinct ExtentId"""
   }
 
+  def generateTableMoveExtentsAsyncCommand(sourceTableName: String, destinationTableName: String, batchSize:Int): String
+  = {
+    s""".move async extents to table $destinationTableName <|
+       .show table $sourceTableName extents with(extentsShowFilteringRuntimePolicy='{"MaximumResultsCount":$batchSize}');
+        $$command_results
+       |  distinct ExtentId"""
```

Comment:
You'll get a list of only unique extent ids.
* Don't you need the operationIds?
* Why do you need to call distinct? where are the duplications?

Reply:
this is one operation 
i dont need distinct anymore - this command changed a while ago - will fix